File: mobile/DisqusSDKDemoIOS/DisqusSDKDemoIOS/Models/Article.swift
// Copyright (c) 2024 Publisher. Licensed under the MIT License.
import Foundation

struct Article {
    let id: String
    let title: String
    let url: String
    let shortName: String
    let apiKey: String

    static let samples: [Article] = [
        Article(
            id:        "ARTICLE_ID",
            title:     "ARTICLE_TITLE",
            url:       "YOUR_URL",
            shortName: "YOUR_SHORT_NAME",
            apiKey:    "YOUR_API_KEY"
        ),
    ]
}

File: mobile/DisqusSDKDemoAndroid/app/src/main/java/com/example/disqussdkdemoandroid/data/Article.kt
package com.example.disqussdkdemoandroid.data

data class Article(
    val id: String,
    val title: String,
    val url: String,
    val shortName: String,
    val apiKey: String,
) {
    companion object {
        val SAMPLES = listOf(
            Article(
                id        = "ARTICLE_ID",
                title     = "ARTICLE_TITLE",
                url       = "YOUR_URL",
                shortName = "YOUR_SHORT_NAME",
                apiKey    = "YOUR_API_KEY"
            )
        )
    }
}
